datajoint/condition.py

```diff
@@ -61,6 +61,13 @@ def append(self, restriction):
             super().append(restriction)
 
 
+class Top:
+    def __init__(self, order_by, limit=None, offset=0):
```

Comment:
The `order_by` argument should also be optional. It's possible to limit the entries without ordering them.

Reply:
What should the first argument be? `limit` makes the most sense to me since `dj.Top(5, order_by='key desc')` seems more intuitive to me than `dj.Top('key desc', limit=5)`.

Also how do we want to handle empty `dj.Top()`s since they have no effect? Should we allow them silently, throw a warning, or throw an error?

Reply:
Just a few thoughts:

- `dj.Top(order_by="KEY")` should order by the primary key -- this has a minimal performance effect.
- Perhaps ordering by the primary key should be default behavior, so `oder_by` defaults to `"KEY"`.
- We could assume a sensible default limit, e.g. `limit=10` 
- Yes, then it may make sense to put limit as the first argument, something like: `__init__(limit=10, *, order_by="KEY", offset=0)`